test/jdk/tools/jlink/JLink20000Packages.java

```diff
@@ -63,32 +67,22 @@ static void javac(String[] args) {
     public static void main(String[] args) throws Exception {
```

Comment:
The:
```java
    private static final ToolProvider JAVAC_TOOL = ToolProvider.findFirst("javac")
            .orElseThrow(() -> new RuntimeException("javac tool not found"));

    static void report(String command, String[] args) {
        System.out.println(command + " " + String.join(" ", Arrays.asList(args)));
    }

    static void javac(String[] args) {
        report("javac", args);
        JAVAC_TOOL.run(System.out, System.err, args);
    }
```
seems no longer needed/used?

(Otherwise the direction seems sensible to me.)